scripts/cluster_network.py

```diff
@@ -426,28 +428,23 @@ def clustering_for_n_clusters(
             f"potential_mode should be one of 'simple' or 'conservative' but is '{potential_mode}'"
         )
 
-    if custom_busmap:
-        busmap = pd.read_csv(snakemake.input.custom_busmap, index_col=0, squeeze=True)
-        busmap.index = busmap.index.astype(str)
-        logger.info(f"Imported custom busmap from {snakemake.input.custom_busmap}")
-    else:
-
-        if snakemake.config["cluster_options"]["alternative_clustering"]:
+    if not isinstance(custom_busmap, pd.Series):
+        if alternative_clustering:
             n, busmap = busmap_for_gadm_clusters(
```

Comment:
In this function we could remove the "n" s output, in the function it seems like it is not modified, by doing so we also remove the "TODO"

Reply:
Done. Thanks for that :)